### DISCUSSION TITLE
Map JSON body

### DISCUSSION THREAD

**Discussion by 5011-5011** [nextgenhealthcare/connect#5893] [Q&A] [answered]
How can I get a value from the body JSON, from a filter?

**Comment by pacmano1** [comment]
Clarify that please.   Use Mirth terms where applicable.

For example:   I am making an HTTP call and get a JSON response back, in the next destination I want to reference the response body of that call.

**Reply by 5011-5011** [reply]
Ok, sorry.
I have a HTTP listening channel. 
I need to take a JSON value from the listener filter, the idea is to validate if the required property exists.
However as the filter is processed before the transformer I can't map it as I usually do in transformers.
I need to take only the property "hl7".

![image](https://github.com/nextgenhealthcare/connect/assets/98954172/c342ae4b-70b9-4c3c-a1a1-7dca2fecdf39)

**Reply by pacmano1** [reply, marked as answer]
![image](https://github.com/nextgenhealthcare/connect/assets/44065187/0f88cd05-def5-48d9-a280-9b36f6249d19)

or:

![image](https://github.com/nextgenhealthcare/connect/assets/44065187/da264ed3-0fca-455b-9a2f-b48f029e4959)

**Reply by 5011-5011** [reply]
Thank you very much! That worked.
